server/src/test/java/org/apache/iotdb/db/query/dataset/groupby/GroupByFillDataSetTest.java

```diff
@@ -42,7 +42,7 @@
   private String[] sqls = {
     "SET STORAGE GROUP TO root.vehicle",
     "CREATE TIMESERIES root.vehicle.d0.s0 WITH DATATYPE=INT32, ENCODING=RLE",
-    "CREATE TIMESERIES root.vehicle.d0.s1 WITH DATATYPE=TEXT, ENCODING=PLAIN",
+    "CREATE TIMESERIES root.vehicle.d0.s1 WITH DATATYPE=INT32, ENCODING=RLE",
```

Comment:
Hi, why change this?

Reply:
SQL:   "select last_value(s0) from root.vehicle.* where s1 > 1 or s0 > 1  group by ([0,20), 1ms) fill (int32[Previous]) order by time desc"    "select last_value(s0) from root.vehicle.* where s1 > 1  group by ([0,20), 1ms) fill (int32[Previous]) order by time desc"    s1 is TEXT